lib/tasks/va_facilities.rake

```diff
@@ -8,142 +8,143 @@ namespace :va_facilities do
+                  street_address_zip_code_extension: vaf["Street Address-Zip Code Extension"].to_s,
+                  county_street_address: vaf["County (Street Address)"],
+                  mailing_address: vaf["Mailing Address"],
+                  mailing_address_city: vaf["Mailing Address-City"],
+                  mailing_address_state: vaf["Mailing Address-State"],
+                  mailing_address_zip_code: vaf["Mailing Address-Zip Code"].to_s,
+                  mailing_address_zip_code_extension: vaf["Mailing Address-Zip Code Extension"].to_s,
+                  county_mailing_address: vaf["County (Mailing Address)"],
+                  station_phone_number: vaf["Station Phone Number"],
+                  station_main_fax_number: vaf["Station Main Fax Number"],
 
                   # these 4 not in VetCenter json schema...
-                  after_hours_phone_number: vaf["After_Hours_Phone_Number"],
-                  pharmacy_phone_number: vaf["Pharmacy_Phone_Number"],
-                  enrollment_coordinator_phone_number: vaf["Enrollment_Coordinator_Phone_Number"],
-                  patient_advocate_phone_number: vaf["Patient_Advocate_Phone_Number"],
+                  after_hours_phone_number: vaf["After Hours Phone Number"],
+                  pharmacy_phone_number: vaf["Pharmacy Phone Number"],
+                  enrollment_coordinator_phone_number: vaf["Enrollment Coordinator Phone Number"],
+                  patient_advocate_phone_number: vaf["Patient Advocate Phone Number"],
 
                   latitude: vaf["Latitude"].to_f,
                   longitude: vaf["Longitude"].to_f,
-                  congressional_district: vaf["Congressional_District"],
+                  congressional_district: vaf["Congressional District"],
                   market: vaf["MARKET"],
                   sub_market: vaf["SUBMARKET"],
                   sector: vaf["SECTOR"],
-                  fips_code: vaf["FIPS_Code"],
-                  rurality: vaf["Rurality_U_Urban_R_Rural_H_Highly_Rural"],
+                  fips_code: vaf["FIPS Code"],
+                  rurality: vaf["Rurality: U=Urban; R=Rural; H=Highly ruralI=Insular"],
                   monday: vaf["Monday"],
                   tuesday: vaf["Tuesday"],
                   wednesday: vaf["Wednesday"],
                   thursday: vaf["Thursday"],
                   friday: vaf["Friday"],
                   saturday: vaf["Saturday"],
                   sunday: vaf["Sunday"],
-                  hours_note: vaf["HoursNote"],
+                  hours_note: vaf["Hours Note"],
                   hidden: hidden
               )
           else
             # update record.....
-            facility = VaFacility.where(station_number: vaf["Station_Number"]).first
+            facility = VaFacility.where(station_number: vaf["Station Number"]).first
+            classification = vaf["Classification"].blank? ? "Unclassified" : vaf["Classification"]
```

Comment:
Instead of creating this variable twice, we could probably just place it below 
```
va_facilities.each do |vaf|
```
 and above
```
if visn.number === vaf["VISN"].to_i && VaFacility.where(station_number: vaf["Station Number"]).empty?
```

Reply:
Good catch Josh.  I moved the line to initialize "facility"..  FYI.. I used a json converter to convert from csv to json... so I'm pretty sure it's valid JSON or line 7 would blow up.  i.e., va_facilities = JSON.parse(file)

Reply:
I was referring to `classification =` regarding the initial comment. Looks like we can do it for both `facility` and `classification`.

Reply:
Got it..